src/Language/Fixpoint/Types/Names.hs

```diff
@@ -44,6 +44,7 @@ module Language.Fixpoint.Types.Names (
   , dropSym
   , headSym
   , lengthSym
+  , mappendSym
```

Comment:
Why is mappend being exported? There are various other exported functions
that add suffixes etc. please use those instead!

On Monday, May 16, 2016, Ranjit Jhala <jhala@cs.ucsd.edu> wrote:

> Yes please don't mess with that API ...
>
> On Sunday, May 15, 2016, Eric Seidel <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
>> In src/Language/Fixpoint/Types/Names.hs
>> <https://github.com/ucsd-progsys/liquid-fixpoint/pull/227#discussion_r63310965>
>> :
>>
>> > @@ -44,6 +44,7 @@ module Language.Fixpoint.Types.Names (
>> >    , dropSym
>> >    , headSym
>> >    , lengthSym
>> > +  , mappendSym
>>
>> I believe we don't export this on purpose. Constructing a Symbol
>> involves hashing the string and performing a table lookup, which is
>> expensive, so we want to keep symbol construction to a minimum. Better to
>> perform all formatting operations on the raw Text and then make a single
>> Symbol at the end.
>>
>> —
>> You are receiving this because you are subscribed to this thread.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/ucsd-progsys/liquid-fixpoint/pull/227/files/019c3bdb49e2988b8a59fbff30cd84b7450b862b..1a6a2b4f034faee428e2bf1c6c08dcbfc1238b9c#r63310965>
>>
>